packages/expo-media-library/android/src/main/java/expo/modules/medialibrary/contracts/DeleteContract.kt

```diff
@@ -0,0 +1,43 @@
+package expo.modules.medialibrary.contracts
+
+import android.app.Activity
+import android.content.ContentResolver
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import android.os.Build
+import android.provider.MediaStore
+import androidx.activity.result.IntentSenderRequest
+import androidx.annotation.RequiresApi
+import expo.modules.kotlin.activityresult.AppContextActivityResultContract
+import expo.modules.kotlin.providers.AppContextProvider
+import java.io.Serializable
+
+class DeleteContract(
+  private val appContextProvider: AppContextProvider
+) : AppContextActivityResultContract<DeleteContractInput, Boolean> {
+  companion object {
+    const val ACTION_INTENT_SENDER_REQUEST =
+      "androidx.activity.result.contract.action.INTENT_SENDER_REQUEST"
+    const val EXTRA_INTENT_SENDER_REQUEST =
+      "androidx.activity.result.contract.extra.INTENT_SENDER_REQUEST"
```

Comment:
Those are also used in `WriteContract`, can you extract them to a separate file?